topics/variant-analysis/tutorials/vir_var0/tutorial.md

```diff
@@ -0,0 +1,618 @@
+      "value": "https://zenodo.org/record/3519268/files/"
+    },
+    {
+      "type": "add_column_concatenate",
+      "target_column_0": 1,
+      "target_column_1": 0
+    },
+    {
+      "type": "remove_columns",
+      "target_columns": [
+        1
+      ]
+    },
+    {
+      "type": "add_column_value",
+      "value": ".fq.gz"
+    },
+    {
+      "type": "add_column_concatenate",
+      "target_column_0": 1,
+      "target_column_1": 2
+    },
+    {
+      "type": "remove_columns",
+      "target_columns": [
+        1,
+        2
+      ]
+    }
+  ],
+  "mapping": [
+    {
+      "type": "list_identifiers",
+      "columns": [
+        0
+      ],
+      "editing": false
+    },
+    {
+      "type": "url",
+      "columns": [
+        1
+      ]
+    }
+  ],
+  "extension": "fastqsanger.gz"
+}
+> > ```
+> {: .tip}
+{: .hands_on}
+
+Once the collection is created and uploaded, you will see a new green box in the history pane as shown in Fig. [2](#figure-2) below:
+
+![Dataset collection](../../images/hiv_collection.png "HIV datasets uploaded as a collection. In this case the collection contains four datasets"){: width="100px"}
+
+# Quality control
+
+## Assessing read quality
+
+Once the data is we will need to assess its quality. There is a number of tutorials here already covering this [subject]({{ site.baseurl }}{% link topics/sequence-analysis/index.md %}), but for continuity we go through QC steps anyway.
+
+> ### {% icon hands_on %} QC: Assessing read quality
+>
+> 1. **FastQC** {% icon tool %} with the following parameters:
+>    - {% icon param-collection %} *"Short read data from your current history"*: `Jair data` (or whatever other name you gave that collection; red arrow in Fig. [3](#figure-3))
+>    - Click **Execute** button
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > Note that **FastQC** produces two output collections:
+>    >
+>    > 1. "Webpage", and
+>    > 2. "Raw Data"   
+>    >
+>    > "Raw data" is what we will be using next.
+>    {: .comment}
+>
+{: .hands_on}
+
+![Fastqc interface](../../images/hiv_fastqc1.png "FastqQC using collection as an input. Note that a collections button ({% icon param-collection %}) was pressed to reveal the collection in the history").
+
+**FastQC** produces an individual report for each dataset. This means that in order to get an idea about quality of the data one needs to click report corresponding to each element of the collection. This may be possible for four datasets but will be very difficult for more than that. Fortunately, there is a tool, **MultiQC** that summarize data for all datasets at once. It takes collection produced by **FastQC** as input and produces a single report. Let apply it our example:
+
+> ### {% icon hands_on %} QC: Summarizing FastQC results
+>
+> 1. **MultiQC** {% icon tool %} with the following parameters:
+>    - *"Which tool was used generate logs?"* : `FastQC` (red arrow in  Fig. [4](#figure-4))
```

Comment:
```suggestion
>    - *"Which tool was used generate logs?"* : `FastQC`
```

Would you mind removing the screenshots? We put quite a lot of effort into removing those, since they really hurt accessibility of the training materials. The parameter name should be sufficient for users.